<ui version="4.0" >
 <class>ScheduleDialog</class>
 <widget class="QDialog" name="ScheduleDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>550</width>
    <height>340</height>
   </size>
  </property>
  <property name="acceptDrops" >
   <bool>false</bool>
  </property>
  <property name="windowTitle" >
   <string>Backup Scheduler</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="labelTitle" >
       <property name="font" >
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Backup Scheduler</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>331</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBoxSchedule" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="title" >
      <string>Schedule options</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="radioButtonDaily" >
        <property name="text" >
         <string>Daily</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="labelTime" >
        <property name="text" >
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QTimeEdit" name="timeEditTime" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat" >
         <string>hh:mm</string>
        </property>
        <property name="time" >
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="labelDate" >
        <property name="text" >
         <string>Day:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="checkBoxMonday" >
        <property name="text" >
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QCheckBox" name="checkBoxFriday" >
        <property name="text" >
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QCheckBox" name="checkBoxTuesday" >
        <property name="text" >
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QCheckBox" name="checkBoxSaturday" >
        <property name="text" >
         <string>Saturday</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QCheckBox" name="checkBoxWednesday" >
        <property name="text" >
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QCheckBox" name="checkBoxSunday" >
        <property name="text" >
         <string>Sunday</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QCheckBox" name="checkBoxThursday" >
        <property name="text" >
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QRadioButton" name="radioButtonMinutesAfterBooting" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Minutes after booting:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QLineEdit" name="lineEditMinutesAfterBooting" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4" >
       <widget class="QCheckBox" name="checkBoxDeleteExtraneous" >
        <property name="text" >
         <string>Delete extraneous files and directories</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>311</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonDaily</tabstop>
  <tabstop>timeEditTime</tabstop>
  <tabstop>checkBoxMonday</tabstop>
  <tabstop>checkBoxTuesday</tabstop>
  <tabstop>checkBoxWednesday</tabstop>
  <tabstop>checkBoxThursday</tabstop>
  <tabstop>checkBoxFriday</tabstop>
  <tabstop>checkBoxSaturday</tabstop>
  <tabstop>checkBoxSunday</tabstop>
  <tabstop>radioButtonMinutesAfterBooting</tabstop>
  <tabstop>lineEditMinutesAfterBooting</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScheduleDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>435</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
